xl2times/transforms.py

```diff
@@ -405,6 +414,21 @@ def process_flexible_import_table(
 
         attribute = "attribute"
         if table.tag != datatypes.Tag.tfm_upd:
+
+            # Check for duplicate DF columns
+            duplicated_cols = [
+                item
+                for item, count in collections.Counter(data_columns).items()
+                if count > 1
+            ]
+            if len(duplicated_cols) > 0:
+                logger.warning(
+                    f"Duplicate data columns in table: {duplicated_cols}.  Dropping first duplicated column. Table: \n{repr(table)}"
+                )
+                # drop duplicate Df columns
+                df = df.loc[:, ~df.columns.duplicated(keep="last")]
+                data_columns = pd.unique(data_columns).tolist()
+
```

Comment:
```suggestion
```

Reply:
@SamRWest let's discuss any changes to treating duplicate columns in an issue first.
For data columns, the current approach should be fine. We may need to do something with duplicate columns specifying indices.

Reply:
Apologies, this change slipped in accidentally when merging from my older branch.  
I've reverted this until we can replicate the correct behaviour here.